Mathlib/RingTheory/Ideal/Span.lean

```diff
@@ -190,13 +189,11 @@ end Semiring
 
 section CommSemiring
 
-variable {a b : α}
-
 -- A separate namespace definition is needed because the variables were historically in a different
 -- order.
 namespace Ideal
```

Comment:
It would seem like this comment no longer applies, and the three `namespace Ideal` sections can be merged...